app/routers/polygon.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel, Field
+from geojson_pydantic import Feature
+
+coordinates = [
+    [-78.990935, 12.437303],
+    [-78.990935, 4.214931],
+    [-66.851117, 4.214931],
+    [-66.851117, 12.437303],
+    [-78.990935, 12.437303],
+]
+
+geojson_polygon = {
+    "type": "Feature",
+    "properties": {},
+    "geometry": {"type": "Polygon", "coordinates": [coordinates]},
+}
```

Comment:
Should we separate this example polygon in other file?

Reply:
What would be the benefit?

Reply:
We decided to keep this example in the polygon file because it is a short example.